packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -414,18 +377,14 @@ export async function deployPreBuilt({
     let nango_config_id: number;
     let provider_config_key: string;
 
-    if (nangoYamlBody) {
```

Comment:
It felt unused/impossible let me know

Reply:
Yeah, don't see any case for this.

Reply:
Looked at this again and I think this PR breaks the admin deploy command https://github.com/NangoHQ/nango/blob/master/packages/cli/lib/services/deploy.service.ts#L79-L99

It is not really used anymore so maybe we should remove that functionality completely?